CMakeLists.txt

```diff
@@ -68,15 +68,15 @@ configure_package_config_file(cmake/${PROJECT_NAME}Config.cmake.in cmake/${PROJE
   INSTALL_PREFIX
     ${CMAKE_INSTALL_PREFIX}
   INSTALL_DESTINATION
-    ${CMAKE_INSTALL_DATADIR}/xrootd/cmake
+    ${CMAKE_INSTALL_LIBDIR}/cmake/xrootd
```

Comment:
Can you please make it `${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}` (i.e. `XRootD`, not `xrootd`)? Thanks.

Reply:
Done.